frontend/src/components/ButtonSelect.css

```diff
@@ -1,20 +1,3 @@
-.smart-hub--button-select-toggle-btn {
```

Comment:
Created a DropdownMenu component that encompasses as much of the press a blue button get a dropdown as possible. The CSS still probably needs a good cleanup